### DISCUSSION TITLE
Best way to do an affine transformation on Euler angles of ebsd data

### DISCUSSION THREAD

**Discussion by jhiscocks** [mtex-toolbox/mtex#711] [Ask Anything]
I have a square EBSD data set which turned out to be sheared during acquisition (i.e. the actual area scanned was a parallelogram).  To correct the spatial ebsd data, I can easily use the affinetrans command.

However, while the affine transformation (affinetrans command) updates the xy coordinates of the data, it doesn’t change the Euler angles.   I need to effectively correct for the fact that the x and y axes were not actually perpendicular during data acquisition.  Rephrased, I need to map the Euler angles for two non-perpendicular xy axes to a perpendicular set of xy axes.

What's the best way to do this?

**Comment by kilir** [comment]
Hi,
I'm wondering if this correction must actually happen on the pattern before indexing takes place.
Cheers,
Rüdiger

**Comment by bjoerneske** [comment]
Dear Jessica

As I see it the geometric correction of Euler angles during aquisition has nothing to do with the geometric distortion of your sample coordinates. The aquisition software does the calibartion based on the pattern center to correct the geometry. The angular distortion might be difficult to trace back since this is not directly related to your sample coordinate distortion.

If your pattern calibrations were done over your scan area I personally think your euler coordinates are ok as the euler calibration does not relate to your trapezoidal sample coordinates but to the orthogonal coordinate system of your SEM. At least this would be the case for the EDAX systems.

Bjørn Eske
